versions.txt

```diff
@@ -2,21 +2,21 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.67.0
+opentelemetry-collector=0.68.0
 
 # Represents the current release of the OpenTelemetry Operator.
-operator=0.67.0
+operator=0.68.0
 
 # Represents the current release of the Target Allocator.
-targetallocator=0.67.0
+targetallocator=0.68.0
 
 # Represents the current release of Java instrumentation.
 # Should match autoinstrumentation/java/version.txt
-autoinstrumentation-java=1.21.0
+autoinstrumentation-java=1.22.1
 
 # Represents the current release of NodeJS instrumentation.
 # Should match value in autoinstrumentation/nodejs/package.json
-autoinstrumentation-nodejs=0.34.0
+autoinstrumentation-nodejs=0.35.0
```

Comment:
```
    logger.go:42: 20:15:15 | instrumentation-nodejs-multicontainer | 2023-01-19 20:12:50 +0000 UTC	Warning	Pod my-deployment-with-sidecar-5f4d86f6d5-h5hfw.spec.initContainers{opentelemetry-auto-instrumentation}		Failed	Failed to pull image "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0": rpc error: code = NotFound desc = failed to pull and unpack image "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0": failed to resolve reference "ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0": ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.35.0: not found	
```

[`v0.34.0`](https://github.com/open-telemetry/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fautoinstrumentation-nodejs) appears to be the most recent version avilable here.

Reply:
Thanks. I have updated now.